src/docs/product/integrations/compliance/vanta-eu/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Vanta EU
+sidebar_order: 1
+redirect_from:
+  - /product/integrations/vanta-eu/
+description: "Learn about Sentry's Vanta integration and how it helps automate security compliance."
+---
+
+Sentry's Vanta EU integration grants Vanta read-only access to verify that Sentry organization members are associated with employees in your organization, which automates security compliance. This integration is maintained and supported by the company that created it. For more details, check out our [Integration Platform documentation](/product/integrations/integration-platform/).
+
+
+<Alert level="info" title="EU Warning">
+
+This integration is meant to integrate with Vanta EU
```

Comment:
```suggestion
This is a Vanta **EU** integration.
```

I'm assuming this warning is here so users don't mistake this for regular Vanta?